plugin/init.lua

```diff
@@ -52,12 +52,15 @@ end
 ---@param file_name string
 ---@param type string
 ---@param opt_name string?
+---@param date_fmt string?
 ---@return string
-local function get_file_path(file_name, type, opt_name)
+local function get_file_path(file_name, type, opt_name, date_fmt)
```

Comment:
Maybe the optional args should be changed to a table with both options, so that you can call the function with a `data_fmt` but without giving an `opt_name`

Reply:
Same is present in the other functions

Reply:
> Maybe the optional args should be changed to a table with both options, so that you can call the function with a `data_fmt` but without giving an `opt_name`

I can make this update later if you'd like. I think it's a better interface. I added another arg instead of a table so it wouldn't be a breaking change. If we switch it to an opts table, users will need to update their configs if they're using `opt_name`.